app/controllers/form_controller.rb

```diff
@@ -374,4 +393,28 @@ def set_managing_organisation_to_assigned_to_organisation?(saledate)
 
     true
   end
+
+  def render_check_errors_page
+    if params[@log.model_name.param_key]["clear_question_id"]
+      question_id = params[@log.model_name.param_key]["clear_question_id"]
+      @log.form.get_question(question_id, @log).page.questions.map(&:id).each { |id| @log[id] = nil }
```

Comment:
Why is this needing to go through all questions on a page?

Reply:
I think I answered it on another PR, but it's basically this:
_ I think some of our pages have additional validations checking that when there are 2 questions on the page, all of the questions have to be answered. So when we choose to clear a single question from a page with 2 questions, we get a validation error. For example, if we have uprn known - yes and a uprn value, when we choose to clear uprn to correct a composite error, the log would go into an invalid state (another example of this where the questions aren’t conditional is income frequency and earnings for lettings)_